app/src/main/java/org/jellyfin/mobile/MainActivity.kt

```diff
@@ -56,13 +59,20 @@ class MainActivity : AppCompatActivity() {
         bindService(Intent(this, RemotePlayerService::class.java), serviceConnection, Service.BIND_AUTO_CREATE)
 
         // Check WebView support
-        if (!isWebViewSupported()) {
+        if (!appPreferences.ignoreWebViewChecks && !isWebViewSupported()) {
```

Comment:
It already checks for ignore here, but there's no way to ignore from the "unsupported" dialog below.. seems like an oversight?

Reply:
I should definitely not work on the app when I'm half asleep 😅